packages/client/src/sharedWishlists/deleteSharedWishlist.ts

```diff
@@ -0,0 +1,22 @@
+import { adaptError } from '../helpers/client/formatError';
+import client from '../helpers/client';
+import join from 'proper-url-join';
+import type { DeleteSharedWishlist } from './types';
+
+/**
+ * Method responsible for update a shared wishlist, synchronizing the prior snapshot with the current state of the related wishlist set.
```

Comment:
Please review this comment, I think you mean "Method responsible dor **deleting**" (and see if the rest makes sense)

Reply:
Thank you for mentioning 
Updated 👍